ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -4,19 +4,20 @@
 //
 
 import UIKit
+import Combine
 
 // MARK: BadgeViewDataSource
 @objc(MSFBadgeViewDataSource)
 open class BadgeViewDataSource: NSObject {
     @objc open var text: String
-    @objc open var style: BadgeView.Style
-    @objc open var size: BadgeView.Size
+    @objc open var style: MSFBadgeViewStyle
```

Comment:
Would this be considered an API break? If so, could you add the label on the PR?